doc/docs.md

```diff
@@ -685,6 +684,70 @@ for i in 0 .. 1000 {
 Note: The above code uses a [range `for`](#range-for) statement and a
 [push operator (`<<`)](#array-operations).
 
+#### Array Types
+
+An array can be of these types:
+| Types        | Example Definition    |
+| ------------ | --------------------- |
+| Number       | `[]int,[]i64`         |
+| String       | `[]string`            |
+| Rune         | `[]rune`              |
+| Boolean      | `[]bool`              |
+| Array        | `[][]int`             |
+| Struct       | `[]MyStructName`      |
+| Channel      | `[]chan`              |
+| Function     | `[]func`              |
```

Comment:
Either a function type has been declared before, then it would be `[]MyFunctionType`
or the function type is specified: `[]fn (int) bool`.